CHANGELOG.md

```diff
@@ -195,3 +195,8 @@ can get data for both organization and organization brand pages.
 * Update edgeType parameter value for the /networkSizes endpoint to be 
   COMPANY_FOLLOWED_BY_MEMBER instead of CompanyFollowedByMember. 
   This changed in the LinkedIn API 202305 release.
+
+## 4.5.1 (Work in progress)
```

Comment:
Merging into master - this needs today's date

Reply:
Thanks David, made these changes here: https://github.com/ebx/ebx-linkedin-sdk/pull/384

Reply:
Thanks for reviewing that, ready to go now